source/extensions/filters/http/proto_api_scrubber/filter_config.cc

```diff
@@ -81,6 +91,49 @@ ProtoApiScrubberFilterConfig::initialize(const ProtoApiScrubberConfig& proto_con
   return absl::OkStatus();
 }
 
+absl::Status ProtoApiScrubberFilterConfig::initializeDescriptorPool(
+    Api::Api& api, const ::envoy::config::core::v3::DataSource& data_source) {
+  Envoy::Protobuf::FileDescriptorSet descriptor_set;
+  auto pool = std::make_unique<Envoy::Protobuf::DescriptorPool>();
+
+  switch (data_source.specifier_case()) {
+  case envoy::config::core::v3::DataSource::SpecifierCase::kFilename: {
+    auto file_or_error = api.fileSystem().fileReadToEnd(data_source.filename());
+    if (!file_or_error.status().ok() || !descriptor_set.ParseFromString(file_or_error.value())) {
+      std::cout << "File_or_Error_Status: " << file_or_error.status() << std::endl;
+      std::cout << "File_or_Error_Status message: " << file_or_error.status().message()
```

Comment:
Seems that this doesn't capture the case where file_or_error is valid (but ParseFromString fails).
Please fix to capture the correct error.